CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ __Notable Changes__
 __Fixed Bugs__
 
 * Fix setting of locale when `current_alchemy_user.language` doesn't return a Symbol (#1097)
+* Fix bug that rendered duplicate nested elements within a cell after copying a parent element.
```

Comment:
Sorry, if I was not clear. We still want this in 3.3-stable, but merged into master. We will cherry-pick it to 3.3-stable then. Could you please update the changelog once again. Sorry for the inconvenience